src/main/java/knightminer/inspirations/building/BuildingClientProxy.java

```diff
@@ -173,10 +185,24 @@ public void onModelBake(ModelBakeEvent event) {
 		replaceBookshelfModel(event, InspirationsBuilding.shelf_rainbow);
 		replaceBookshelfModel(event, InspirationsBuilding.shelf_tomes);
 
-		if(InspirationsBuilding.enlightenedBush != null) {
-			ResourceLocation location = InspirationsBuilding.enlightenedBush.getRegistryName();
-			replaceTexturedModel(event, new ModelResourceLocation(location, ""), "leaves", true);
-		}
+		replaceBushModel(event, InspirationsBuilding.whiteEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.redEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.blueEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.greenEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.rainbowEnlightenedBush);
+		replaceBushModel(event, InspirationsBuilding.christmasEnlightenedBush);
+	}
+
+	private static void replaceBushModel(ModelBakeEvent event, BlockEnlightenedBush bush) {
+		ResourceLocation bushLoc = Objects.requireNonNull(bush.getRegistryName());
+		replaceTexturedModel(event, new ModelResourceLocation(bushLoc, ""), "leaves", false);
+
+		ResourceLocation itemLoc = new ModelResourceLocation(bushLoc, "inventory");
+		IModel model = ModelLoaderRegistry.getModelOrLogError(itemLoc,"Error loading item model for " + itemLoc);
+		IBakedModel standard = event.getModelRegistry().get(itemLoc);
+		IBakedModel itemModel = new TextureModel(standard, model, DefaultVertexFormats.ITEM, "texture", true);
```

Comment:
Just make this a parameter on `replaceTextureModel` for `DefaultVertexFormats.ITEM` (boolean or vertex format) instead of copying all the code over